vignettes/lintr.Rmd

```diff
@@ -216,7 +216,16 @@ If you want to use all available linters *except* a few, you can exclude them us
 linters: linters_with_defaults(
     commented_code_linter = NULL,
     implicit_integer_linter = NULL,
-    defaults = linters_with_tags(tags = NULL)
+    defaults = all_linters()
```

Comment:
I wouldn't show this way to write the config. It's unnecessarily verbose.

Reply:
True. Removed it.